HIP-Basic/texture_management/CMakeLists.txt

```diff
@@ -57,23 +57,36 @@ endif()
 
 list(APPEND CMAKE_PREFIX_PATH "${ROCM_ROOT}")
 
-add_executable(${example_name} main.hip)
-# Make example runnable using ctest
-add_test(${example_name} ${example_name})
-
-# Temporary workaround: a known bug prevents the example from executing succesfully
-# if multiple GPUs are visible 
-set_tests_properties(
-    ${example_name}
-    PROPERTIES ENVIRONMENT "HIP_VISIBLE_DEVICES=0"
+# Check if GPU architecture is supported
+list(REMOVE_DUPLICATES CMAKE_HIP_ARCHITECTURES)
+message(
+    STATUS
+    "texture_management - CMAKE_HIP_ARCHITECTURES: ${CMAKE_HIP_ARCHITECTURES}"
 )
 
-set(include_dirs "../../Common")
-if(GPU_RUNTIME STREQUAL "CUDA")
-    list(APPEND include_dirs "${ROCM_ROOT}/include")
-endif()
+if(NOT CMAKE_HIP_ARCHITECTURES MATCHES "gfx94*")
```

Comment:
If we don't add the folder on `HIP-Basic/CMakeLists.txt` this change does not seem necessary IINM. I would suggest either keeping this and removing the skipping code on `HIP-Basic/CMakeLists.txt` or vice-versa.